src/amount.h

```diff
@@ -17,7 +17,7 @@ static const CAmount CENT = 1000000;
 /** No amount larger than this (in satoshi) is valid.
```

Comment:
still remain satoshi.

Reply:
ok
for 'bitcoin' in every README.md i will change to 'Tapyrus' also ?